### PR TITLE
Test. Check frontend project PW

### PR DESCRIPTION
**Pull request by juanjofgliferay**

### CODE REVIEW

**Comment by juanjofgliferay** [comment]
ci:test:frontend

File: test.properties [hunk 1]
```diff
@@ -5610,6 +5616,12 @@ test.batch.run.property.query[functional-orcllinux9-tomcat90-mysql57-jdk8]=\
     # Frontend Infra
     #
 
+    playwright.test.project[playwright-js-tomcat90-mysql57-jdk8][frontend]=\
```
Comment (juanjofgliferay) [inline test.properties:5619]: @dsanz It seems that this is what we need to include PW test in the Testray report when runnint `ci:test:frontend`
We might need to add `client-extensions-web`

https://testray-old.liferay.com/reports/production/logs/2024-07/test-1-37/test-portal-acceptance-pullrequest(master)/17339/playwright-js-tomcat90-mysql57-jdk8/0/0/playwright-report/index.html

image

**Comment by juanjofgliferay** [comment]
ci:test:frontend

File: test.properties (hunk 1 above)
Comment (dsanz) [reply test.properties:5625]: makes sense. We added it already for relevant [in the "local" test.properties](https://github.com/liferay/liferay-portal/blob/9b19e94893a957c85530c83d0ee15e8434000456/modules/apps/frontend-data-set/test.properties#L5-L7). Idk what's the _right_ place if that thing exists.

**Comment by dsanz** [comment]
Closing due to inactivity. If this is not ok, feel free to reopen

**Pull request closed without merging**